### ISSUE TITLE
See whether .navigation--major and .section-nav blocks can be combined

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/vets-website#1952]
The two patterns are very similar. `.navigation--major` exists on the home and 404 pages. 

Blocked by #1672 / #1951

**Comment by webinista** [comment]
`.section-nav` isn't used anywhere in the CSS as a selector. We can leave it the way it is for now, since `navigation--major` is not an actual pattern, just a styling hook for consistent padding. Revisit when #1676 is done. 

(Won't close, but this issue is super low priority.)
